frontend/src/v5/ui/controls/selectWithLabel/selectWithLabel.component.tsx

```diff
@@ -23,11 +23,9 @@ export interface SelectWithLabelProps extends SelectProps {
 }
 
 export const SelectWithLabel = ({ required = false, helperText, label, ...props }: SelectWithLabelProps) => (
-	<FormControl required={required} disabled={props.disabled} error={props.error}>
+	<FormControl required={required} disabled={props.disabled} error={props.error} className={props.className}>
 		<InputLabel id={`${props.name}-label`}>{label}</InputLabel>
-		<Select
-			{...props}
-		/>
+		<Select {...props} />
```

Comment:
This could bring problems,
classname/error/disabled should only be passed to FormControl . Otherwise the select and form control could be affected by the same class resulting in weird styling. 
If there's a need to style the select within `SelectWithLabel`, I think that this should work:
 
```
const myStyledSelectWithLabel = styled(SelectWithLabel)`
${Select} {
   some css code...;
}
`
```

Reply:
The disabled and error were already like this. Should I remove them from the Select's props?